src/Components/search.tsx

```diff
@@ -21,6 +21,7 @@ const CustomSearch = ({
   const clearTextInput = () => {

```

Comment:
You don't have to use "Ref" to clear the search text; simply set the state to empty when the clear icon is pressed.